Bag Totals, Bug Fixes, BinanceUS

HojouFotytu opened the pull request:
- created Monitor strategy shortcut "SOM" for new PT status: "config som enabled"
- fixed Monitor PBL sorting, broken by New PT status "config som enabled"
- Changed "Value" headers to "Cost" on Bag lists, since the value represents total cost, not current value.
- Added total cost and profit % on the bottom of dashboard bag list 
- Fixed broken links to PT wiki on Settings > Analyzer page.

djbadders commented on Monitor/Pages/_get/DashboardTop.cshtml:
```diff
@@ -147,6 +147,11 @@
 
+                // Aggregate totals
+                Model.TotalBagCost = Model.TotalBagCost + dcaLogEntry.TotalCost;
+                double TradingFee = (dcaLogEntry.Amount * dcaLogEntry.CurrentPrice) * 0.002;
```
Are all exchanges charging a 0.2% fee?

HojouFotytu replied:
- Updated with fees per/exchange.  Good catch.  ;)

djbadders commented:
Look like a decent set of changes, nice work. One small comment to consider.

HojouFotytu commented:
- Added exchange: BinanceUS
- Added exchange name to "Market Trends" charts
- Added exchange name to Settings > General > Application
- Fixed code for fees per exchange

djbadders commented on Monitor/Pages/_get/DashboardTop.cshtml:
```diff
@@ -149,7 +149,23 @@
+                }
+                if (Model.PTMagicConfiguration.GeneralSettings.Application.Exchange.Equals("Poloniex")) {
+                  ExchangeFee = 0.0025;
+                }
```
Neater as a switch case block but will work :)

HojouFotytu replied:
Good point.  Changed.
